experimental/opencv-text-recognition/text_recognition.py

```diff
@@ -10,12 +10,25 @@
 import cv2
 import os
 from tqdm import tqdm
+
+
 def sort_boxes(boxes):
-    #TODO: sort boxes so they represent the position of them on page
+    #TODO: sort boxes by they location in the image
+    pass
+
+def resize_img(orgH, org, width):
```

Comment:
Please use typing to improve the readability of code. This should be:

`def resize_img(orgH: int, org: int, width: int) -> None:`

If you need a list/dict/set, these are available in the module `typing`: `def sort(List[int]) -> List[int]: ...`

Also, the names of the variables are not very readable.